db/migrate/20161128200400_update_quotes.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateQuotes < ActiveRecord::Migration
+  def change
+    change_column_null :quotes, :anime_id, false
+    change_column_null :quotes, :user_id, false
+    change_column_null :quotes, :content, false
+
+    add_foreign_key :quotes, :users
+    add_foreign_key :quotes, :anime
+    add_foreign_key :quotes, :characters
```

Comment:
so I should be doing something like:
`Quote.where(media_id: nil).destroy_all` (and the same for content, and media_type?)